### PR TITLE
Feature/feedin split

### LINKED ISSUE #135: Separate feed-in by source
In the resulting json the amount of energy fed into the grid should be saved. Here, a distinction between local energy power sources (e.g. PV), stationary batteries and V2G should be made. In the json "feed-in energy" therefore should have three kWh-values for these three sources.
The existing "feed-in energy" (line 261 in report.py) should be renamed to "local power generation"

### PR DESCRIPTION
**Pull request by j-ti**
Fixes #135

**Changes proposed in this pull request**:
- splits actual feed-in to grid into asset origins, i.e. PV, V2G and battery
  - [x] Adding separate function
  - [x] Adding values to result csv file
  - [x] Add Aggregate values to result json file and adapt example files

The following steps were realized (required):
- [x] Correct linting with `flake8 path_to_script`
- [x] Check if tests pass locally with `python -m pytest tests/`
    - [x] Adapt csv example file
    - [x] create separate branch to fix untouched results and merge
      - generate schedule: https://github.com/rl-institut/spice_ev/tree/feature/merge-dev-wording
      - other tests: https://github.com/rl-institut/spice_ev/tree/fix/example-test-wording

Also the following steps were realized (if applies):
- [x] Write docstrings to your code
- [x] Explain new functionalities in readthedocs 
  - [x] required if docstring is present? => yes
  - [x] update simulating spice ev ... table 
- [x] Write test(s) for new patch of code
- [x] Check building of documentation with `doc/make html`

Further TODO:
- [x] adapt for changing key and variable names in #160

### CODE REVIEW

**Comment by j-ti** [comment]
Preliminary results as below:
spice_ev-feedin_split
Todos:
- [x] feetin values should be negative

File: spice_ev/report.py [hunk 1]
```diff
@@ -424,6 +473,14 @@ def aggregate_timeseries(scenario, gcID):
     if hasSchedule:
         # fixed loads (e.g., building)
         header += ["schedule [kW]", "window signal [-]"]
+    # Feed-in to grid per asset
+    print(hasFeedinComponents)
```
Comment (jakobgemassmer) [inline spice_ev/report.py:477]:
```suggestion

```

File: spice_ev/report.py [hunk 2]
```diff
@@ -327,6 +327,54 @@ def aggregate_local_results(scenario, gcID):
...
+        cs_discharge,
+        accumulated
+    ) if accumulated > 0 else 0
+    # Prio 3: If the remaining feed-in to grid exists,
+    # it originates from the battery, as there is no further asset, that operates as source
+    battery_feedin = (
+        accumulated - v2g_feedin
+    ) if (accumulated - v2g_feedin) > 0 else 0
+    return [
+        - round(generated_feedin, round_to_places),
```
Comment (jakobgemassmer) [inline spice_ev/report.py:372]:
```suggestion
        round(generated_feedin, round_to_places),
```

File: spice_ev/report.py (hunk 1 above)
Comment (stefansc1) [inline spice_ev/report.py:477]: remove print

**Review by stefansc1 — CHANGES_REQUESTED** [review]

File: examples/output/simulation.csv [hunk 3]
```diff
@@ -1,98 +1,98 @@
...
-89,2020-01-01 22:15:00,42,0,0,0,0,0,7,0,1,0,0,0,0
-90,2020-01-01 22:30:00,42,0,0,0,0,0,7,0,1,0,0,0,0
-91,2020-01-01 22:45:00,42,0,0,0,0,0,7,0,1,0,0,0,0
-92,2020-01-01 23:00:00,42,0,0,0,0,0,7,0,1,0,0,0,0
-93,2020-01-01 23:15:00,42,0,0,0,0,0,7,0,1,0,0,0,0
-94,2020-01-01 23:30:00,42,0,0,0,0,0,7,0,1,0,0,0,0
-95,2020-01-01 23:45:00,42,0,0,0,0,0,7,0,1,0,0,0,0
-96,2020-01-02 00:00:00,42,0,0,0,0,0,7,0,1,0,0,0,0
\ No newline at end of file
+timestep,time,price [EUR/kWh],grid supply [kW],fixed load [kW],local generation [kW],flex band min [kW],flex band base [kW],flex band max [kW],generated feed-in [kW],V2G feed-in [kW],sum CS power [kW],# occupied CS [-],# CS in use [-],C1 [kW],C2 [kW],C3 [kW]
```
Comment (jakobgemassmer) [inline examples/output/simulation.csv:1]:
```suggestion
timestep,time,price [EUR/kWh],grid supply [kW],fixed load [kW],local generation [kW],flex band min [kW],flex band base [kW],flex band max [kW],generation feed-in [kW],V2G feed-in [kW],sum CS power [kW],# occupied CS [-],# CS in use [-],C1 [kW],C2 [kW],C3 [kW]
```

**Review by jakobgemassmer — CHANGES_REQUESTED** [review]
add feed-in split by source to resulting json (also add in example files).

File: spice_ev/report.py [hunk 4]
```diff
@@ -399,6 +447,7 @@ def aggregate_timeseries(scenario, gcID):
     hasFixedLoads = any(scenario.fixedLoads)
     hasSchedule = any(s is not None for s in scenario.gcPowerSchedule[gcID])
     hasBatteries = sum([b.parent == gcID for b in scenario.components.batteries.values()])
+    hasFeedinComponents = [any(scenario.localGenerationPower), True, bool(hasBatteries)]
```
[the remark this answers is not in the document]
Comment (stefansc1) [reply spice_ev/report.py:450]: suggested change:
```
hasV2G = any([v.vehicle_type.v2g for v in scenario.components.vehicles.values()])
hasBatteries = any([b.parent == gcID for b in scenario.components.batteries.values()])
hasFeedinComponents = [any(scenario.localGenerationPower), hasV2G, hasBatteries]
```

File: spice_ev/report.py [hunk 5]
```diff
@@ -327,6 +327,54 @@ def aggregate_local_results(scenario, gcID):
     return json_results
 
 
+def split_feedin(grid, generation, cs_sum, round_to_places):
```
[the remark this answers is not in the document]
Comment (j-ti) [reply spice_ev/report.py]: also added 3 as default parameter

**Pull request merged**